src/Inventory.zig

```diff
@@ -1414,6 +1569,58 @@ pub const Command = struct { // MARK: Command
 			};
 		}
 	};
+
+	const AddHealth = struct { // MARK: AddHealth
+		source: ?*main.server.User,
+		health: f32,
+		cause: main.game.DamageType,
+		previous: f32,
```

Comment:
Previous health should not be needed here.
It should be calculated when executing the base operation.